src/env/builtins/lredir.c

```diff
@@ -543,6 +482,12 @@ int lredir_main(int argc, char **argv)
       return(0);
     }
 
+    /* check the MFS redirector supports this DOS */
+    if (!isInitialisedMFS()) {
+      printf("Unsupported DOS version\n");
```

Comment:
This doesn't look right.
How about emufs.sys to print this msg
and lredir2 to print something like
missing emufs.sys?